docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -10,371 +9,175 @@ to save power and free CPU resources.
-|1|2|1|1|1|1|384|240|240|240|240|240|170|113|
-|1|3|1|1|1|1|128|240|240|240|240|240|   |   |
-|1|3|1|1|1|1|256|240|240|240|240|240|   |   |
-|1|3|1|1|1|1|384|240|240|240|240|240|   |   |
-|2|1|1|1|1|1|128|192|192|192|192|192|192|192|
-|2|1|1|1|1|1|256|192|192|192|192|192|192|170|
-|2|1|1|1|1|1|384|192|192|192|192|192|170|113|
-|2|2|1|1|1|1|128|193|193|193|193|193|193|129|
-|2|2|1|1|1|1|256|193|193|193|193|193|193|129|
-|2|2|1|1|1|1|384|193|193|193|193|193|170|113|
-|3|1|1|1|1|1|128|128|128|128|128|128|128|85|
-|3|1|1|1|1|1|256|128|128|128|128|128|128|85|
-|3|1|1|1|1|1|384|128|128|128|128|128|128|85|
-|3|3|1|1|1|1|128|130|130|130|130|87 |   |  |
-|3|3|1|1|1|1|256|130|130|130|130|87 |   |  |
-|3|3|1|1|1|1|384|130|130|130|130|87 |   |  |
-|4|1|1|1|1|1|128|96|96|96|96|96|96|64|
-|4|1|1|1|1|1|256|96|96|96|96|96|96|64|
-|4|1|1|1|1|1|384|96|96|96|96|96|96|64|
-|5|1|1|1|1|1|128|76|76|76|76|76|51|51|
-|5|1|1|1|1|1|256|76|76|76|76|76|51|51|
-|5|1|1|1|1|1|384|76|76|76|76|76|51|51|
-|6|1|1|1|1|1|128|64|64|64|64|64|42|21|
-|6|1|1|1|1|1|256|64|64|64|64|64|42|21|
-|6|1|1|1|1|1|384|64|64|64|64|64|42|21|
-|7|1|1|1|1|1|128|54|54|54|54|54|36|18|
-|7|1|1|1|1|1|256|54|54|54|54|54|36|18|
-|7|1|1|1|1|1|384|54|54|54|54|54|36|18|
-
-
-> **NOTE**:  The above limitations only apply to the new hardware 2D convolution operation.  When possible, the Intel® GNA plugin graph compiler flattens 2D convolutions so that the second generation Intel® GNA 1D convolution operations (without these limitations) may be used.  The plugin will also flatten 2D convolutions regardless of the sizes  if GNA 2.0 compilation target is selected (see below).
-
-## Intel® GNA Forward and Backward Compatibility
-
-In the general case, there is no guarantee that a model compiled for GNA 2.0 will run on GNA 3.0, or vice versa.
-
-However, in most cases, networks compiled for GNA 2.0 will run as expected on GNA 3.0, although the performance may be worse compared to the case when a network is compiled specifically for the latter.  The exception is networks with convolutions with the number of filters greater than 8192 (see the <a href="#models-and-layers-limitations">Models and Layers Limitations</a> section).
-
-Networks compiled for GNA 3.0 should run on GNA 2.0 with incompatible layers emulated on CPU.
-
-You can use the following options `KEY_GNA_EXEC_TARGET` and `KEY_GNA_COMPILE_TARGET` options  to check interoperability (see the <a href="#supported-configuration-parameters">Supported Configuration Parameters</a> section below):
+### Automatic QoS Feature on Windows*
+
+Starting with 2021.4.1 release of OpenVINO and 03.00.00.1363 version of Windows* GNA driver, a new execution mode `ov::intel_gna::ExecutionMode::HW_WITH_SW_FBACK` is introduced
+to assure that workloads satisfy real-time execution. In this mode, the GNA driver automatically falls back on CPU for a particular infer request
+if the HW queue is not empty, so there is no need for explicitly switching between GNA and CPU.
```

Comment:
In this mode, the GNA driver automatically falls back on CPU for a particular infer request. If the HW queue is not empty, there is no need for explicit switching between GNA and CPU.